### PR TITLE
Oleksiy profile update proposal

### PR DESCRIPTION
**Pull request by ababich**

### CODE REVIEW

**Comment by meg-gutshall** [comment]
Thank you @ababich! I'll take a look at this shortly!

**Comment by designbygia** [comment]
@ababich Thanks for this bio write-up! Would you be able to share a little bit about your work experience outside of RfG to create a second paragraph, similar to the other bios on the website? Thanks!

**Comment by meg-gutshall** [comment]
@ababich You can refer to [Sean's profile](https://raw.githubusercontent.com/rubyforgood/rubyforgood.org/gh-pages/all-collections/_our-team/sean-marcia.md) as an example for what @designbygia is talking about.

Also please make sure there are no empty lines between each `- paragraph:` to ensure that the info renders correctly. Thank you!

File: all-collections/_our-team/oleksiy-babich.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+first_name: Oleksiy
+last_name: Babich
+list_order: F
+position: Director of Technology/Infrastructure
+photo: /assets/img/people/oleksiy-babich.png
+email: oleksiy@oleksiy.od.ua
+
+write_up:
+  - paragraph: "Oleksiy Babich, pronouns he, him, his, serves as the Director of Technology/Infrastructure. Oleksiy works on project design and execution, best development and delivery practices to help team leads increase number of contributions, improve quality, security and simplify maintenance. He also provides leadership, thought partnership, and technical support to teams and technical leads on specific software projects, which includes supporting the work of technical leads and product managers, contributing to the architectural design and development of projects, and managing support for the entire life cycle of projects."
+
```
Comment (ababich) [inline all-collections/_our-team/oleksiy-babich.md:13]:
```suggestion

- paragraph: Oleksiy worked with several multi-national enterprises and startup project, as well as contributed to DiaperBase project, Fast JSON API gem and other initiatives. His current focus is on cloud architectures, security, and operational and development excellence.
```

**Comment by ababich** [comment]
@designbygia and @meg-gutshall - thanks a lot for the feedback, I added proposed changes

**Comment by meg-gutshall** [comment]
Hey @ababich! There was one weird indentation issue that messed up the render so I pushed the paragraph into place and we're good to go! Thank you for your contribution!!

**Pull request merged**